docs/README.md

```diff
@@ -12,10 +12,10 @@ associated tools first:
 $ sudo pip install sphinx sphinx-autobuild sphinx_rtd_theme livereload==2.2.0
```

Comment:
Hey @kadamwhite! Thanks so much for the PR for this! 💖

I've just been testing this on a new laptop which is always nice as I can find out what dependencies I'm missing!

Could you please change this line to:
`sudo -H pip install Sphinx sphinx-autobuild sphinx_rtd_theme imagesize livereload==2.2.0`

I had some file permisson issues and was missing `imagesize` on this new computer and also `sphinx-build` wasn't found but changing it to `Sphinx` seemed to fix that for me.